config/coordinator.default.json

```diff
@@ -13,9 +13,27 @@
     "UpdateIntervalMs": 2000,
     "Queries": [
       {
-        "Description": "Request duration",
+        "Description": "Average request duration",
         "Query": "rate(mattermost_http_request_duration_seconds_sum[1m])/rate(mattermost_http_request_duration_seconds_count[1m])",
-        "Threshold": 0.2,
+        "Threshold": 0.1,
+        "Alert": true
+      },
+      {
+        "Description": "Average request duration",
+        "Query": "rate(mattermost_api_time_sum[1m])/rate(mattermost_api_time_count[1m])",
```

Comment:
**Note**

This is actually wrong. These updated metrics do actually return the time/count for each API handler so we need to sum them to get the value we are looking for. Otherwise we'll just consider the first we find in the resulted vector (:man_facepalming:).